ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/bfd-interfaces.j2

```diff
@@ -13,7 +13,8 @@
 {%                 set interval = ethernet_interfaces[ethernet_interface].bfd.interval %}
 {%                 set min_rx = ethernet_interfaces[ethernet_interface].bfd.min_rx %}
 {%                 set multiplier = ethernet_interfaces[ethernet_interface].bfd.multiplier %}
-| {{ ethernet_interface }} | {{ interval }} | {{ min_rx }} | {{ multiplier }} |
+{%                 set echo = ethernet_interfaces[ethernet_interface].bfd.echo | arista.avd.default("False") %}
```

Comment:
Please set the default to `'-'` since the interface modes may be affected by global settings (today or in the future). Thanks

Reply:
Great feedback!  This fixed.  Thanks!